src/sentry/api/endpoints/organization_plugins_configs.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+import six
+
+from sentry.plugins.base import plugins
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.plugin import PluginSerializer
+from sentry.models import ProjectOption, Project
+
+
+# This endpoint is similar to OrganizationPluginsEndpoint
+# Eventually we can replace that one with this
+class OrganizationPluginsConfigsEndpoint(OrganizationEndpoint):
+    def get(self, request, organization):
+        # only load plugins that have configuration
+        # plugins that can't be configured shouldn't be shown
+        all_plugins = dict([(p.slug, p) for p in plugins.plugins_with_configuration()])
+
+        # can specify specific plugins to load
+        if "plugins" in request.GET:
+            desired_plugins = request.GET.getlist("plugins")
+        else:
+            desired_plugins = all_plugins.keys()
+
+        # find all the keys that will tell us if a plugin is enabled OR configured
```

Comment:
I feel like maybe this could be a little clearer (this might be too wordy i dk up to you)
```python
# `keys_to_check` are the ProjectOption keys that tell us if a plugin is enabled (e.g. `plugin:enabled`) or are 
# configured properly, meaning they have the required information - plugin.required_field - needed for the
# plugin to work (e.g.`plugin:auth_key`)
```